src/test/account-utils.ts

```diff
@@ -18,10 +18,86 @@ export const mockedAccount: Account = {
 	identities: LOGGED_USER.identities
 };
 
-export function setupAccountStore(account = mockedAccount): void {
+export function setupAccountStore(
+	account = mockedAccount,
+	accountSettingsPrefs: AccountSettingsPrefs = {}
+): void {
 	useAccountStore.setState(
 		produce((state) => {
 			state.account = account;
+			state.settings.attrs.zimbraIdentityMaxNumEntries = 20;
+			state.settings.prefs = accountSettingsPrefs;
 		})
 	);
 }
+
+export function createAccount(name: string, id: string, identities: Array<Identity>): Account {
+	return {
+		name,
+		rights: { targets: [] },
+		signatures: { signature: [] },
+		id,
+		displayName: '',
+		identities: {
+			identity: identities
+		}
+	};
+}
+
+export function createIdentityAttrs(
+	zimbraPrefIdentityId: IdentityAttrs['zimbraPrefIdentityId'],
+	zimbraPrefIdentityName: IdentityAttrs['zimbraPrefIdentityName'],
+	zimbraPrefFromAddress: IdentityAttrs['zimbraPrefFromAddress'],
+	zimbraPrefFromDisplay: IdentityAttrs['zimbraPrefFromDisplay'] = ''
+): IdentityAttrs {
+	const _attrs: IdentityAttrs = {
+		zimbraPrefReplyToEnabled: 'FALSE',
+		zimbraPrefFromDisplay,
+		zimbraPrefFromAddressType: 'sendAs',
+		zimbraPrefIdentityId,
+		zimbraPrefIdentityName,
+		zimbraPrefFromAddress
+	};
+
+	return _attrs;
+}
+
+export function createDefaultIdentity(
+	zimbraPrefIdentityId: Required<
+		Pick<IdentityAttrs, 'zimbraPrefIdentityId'>
+	>['zimbraPrefIdentityId'],
```

Comment:
Instead of picking a required, we can use directly NonNullable
```suggestion
	zimbraPrefIdentityId: NonNullable<IdentityAttrs['zimbraPrefIdentityId']>,
```